data_model/1.1/clusters/FlowMeasurement.xml

```diff
@@ -66,7 +66,7 @@ Davis, CA 95616, USA
   <clusterIds>
     <clusterId id="0x0404" name="Flow Measurement"/>
   </clusterIds>
-  <classification hierarchy="base" role="application" picsCode="FLW" scope="Endpoint"/>
+  <classification hierarchy="base" role="application" picsCode="FLW" scope="Type 2 (server to client)"/>
```

Comment:
same here - this scope is very odd and would need to be changed to be usable by the scripts.